Update delete.md (#1621)
platformsh/platformsh-docs: docs/src/administration/web/delete.md

* Hover the gear icon on the project and select "Delete".
* Confirm your intent to delete the project.

{{< note >}}
Only the "Owner" of a project has the ability to delete that project. 

> {{< /note> }}

Reviewer: ```suggestion
{{< /note >}}
```

Reviewer: Close the shortcode

* Hover the gear icon on the project and select "Delete".
* Confirm your intent to delete the project.

> Please note that currently only the owner of a project can delete it.

Reviewer: ```suggestion
{{< note >}}
Only the "Owner" of a project has the ability to delete that project. 
{{< /note> }}
```

Reviewer: Resubmit as a note, trigger rebuild.
